packages/KLabel/__snapshots__/KLabel.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`KLabel renders a tooltip when \`help\` is provided 1`] = `<label class="k-input-label" help="This is a tooltip">Full Name</label>`;
```

Comment:
This snapshot doesn't provide as much confidence as I'd like, but the current version of `@vue/test-utils` does not support stubs. Upgrading to `v1.2.0` allows for a much more complete snapshot with `stubs` like so:

```
   <label class="k-input-label">
      <kooltip-stub label="This is a tooltip">Full Name <kicon-stub icon="help" height="16" width="16"></kicon-stub>
      </kooltip-stub>
    </label>
```

However, upgrading to 1.2.0 broke a lot of existing tests that I do not have the context to fix at this time